liquibase-service/src/main/resources/db/odse/routines/010-sp_investigation_event-002.sql

```diff
@@ -767,12 +784,30 @@ BEGIN
                            on results.contact_inv_status_cd = cvg2.code and cvg2.code_set_nm = 'PHC_IN_STS'
                  LEFT OUTER JOIN nbs_odse.dbo.case_management cm WITH (NOLOCK) on results.public_health_case_uid = cm.public_health_case_uid
                  LEFT JOIN
-             (SELECT DISTINCT act_uid       AS                                                 nac_page_case_uid,
-                              MAX(last_chg_time) AS                                                  nac_last_chg_time,
-                              MIN(add_time)      AS                                                 nac_add_time,
-                              MAX(CASE WHEN type_cd = 'PerAsReporterOfPHC' THEN entity_uid END) person_as_reporter_uid,
-                              MAX(CASE WHEN type_cd = 'HospOfADT' THEN entity_uid END)          hospital_uid,
-                              MAX(CASE WHEN type_cd = 'OrgAsClinicOfPHC' THEN entity_uid END)   ordering_facility_uid
+             (SELECT DISTINCT
+                act_uid  AS  nac_page_case_uid,
+                MAX(last_chg_time) AS                                                  nac_last_chg_time,
```

Comment:
Can we validate that the min/max for the fact table is the same across all the entities? Is it possible that they might have different add or last change times?

Reply:
as discussed will keep the condition as is and proceed.